### ISSUE TITLE
Unable to see tooltips for extra content added in the react component

### ISSUE THREAD

**Issue by amndevil687** [wwayne/react-tooltip#318]
I have a react component to render search results.
Initially, I have just 50 results. So I mount 'results' component with 50 results.
<div id='results'>
   <div id='result1' data-tip="result1"></div>
          ..... (till 50 results)
   <ReactTooltip/>
</div>

Now I receive more 30 results. So I update my results component on similar lines.

The issue is, I now only see tooltips for first 50 results and not for the last 50 results.

Please help.

**Comment by myfoxtail** [comment]
I've got the same issue!

**Comment by myfoxtail** [comment]
@amndevil687 try this

`componentDidUpdate() {
  ReactTooltip.rebuild();
}`

It works for me.

**Comment by mjurczyk** [comment]
Unfortunately, that solution doesn't apply if you'd try to use `react-tooltip` in a functional component 😕

**Comment by achang28** [comment]
From the API documentation, given the explanation about rebinding the target location (i.e. where the tooltip will pop-up from), it seems like .rebuild() simply refreshes the Tooltip instance to recognize the most updated values from that target DOM element...

**Comment by avnersorek** [comment]
possible duplicate of https://github.com/wwayne/react-tooltip/issues/354

will this be fixed with https://github.com/wwayne/react-tooltip/pull/273 ?

**Comment by aronhelser** [comment]
I agree this is probably a duplicate of #354 . But I don't think the body-events fix will help - that's just about  how events are responded to, not which elements the tooltip will pay attention to. But it might be worth testing...
